src/mo_frontend/bi_match.ml

```diff
@@ -87,7 +87,7 @@ let bi_match_subs scope_opt tbs subs typ_opt =
     rel := SS.add (t1, t2) !rel;
     match t1, t2 with
     | Pre, _ | _, Pre ->
-      Some inst
+      Some inst (* TODO: assert false? *)
```

Comment:
Reconsider?

Reply:
This is unrelated to the PR, but I noticed this should probably assert.